### DISCUSSION TITLE
Unclear where/what the Java console is

### DISCUSSION THREAD

**Discussion by drtobybrown** [idia-astro/carta-python#21] [Q&A]
When the encountering an error in the scripting interface, the only information given is 

> Error encountered while executing snippet. See JavaScript console for details.

I'm not sure where or what the JavaScript console is but would it not be more helpful to have an ipython console + text / script editor (e.g., such as Jlab)?

Cheers
Toby

**Comment by veggiesaurus** [comment]
The JavaScript console is built into the browser. Are you using the electron app?

Are you using the code snippets feature in the 3.0 feature, or scripting via carta-python?

**Comment by confluence** [comment]
Hi, Toby

You should only be seeing that error if you're using the JavaScript snippet API through the frontend. This repository is for the Python scripting wrapper, a different interface which has some of the same functionality.

Could you please summarise what steps you followed when you saw that error? Have you seen the [documentation for installing and using the wrapper](https://carta-python.readthedocs.io/)?
